commit a3385db5a4318bc65789367585d4988fcc2b5033
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Mon Oct 29 15:58:14 2018 +0000

    staging: bcm2835-camera: Fix multiple assignments should be avoided
    
    Clear checkpatch complaints of "multiple assignments should be avoided"
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
index 44a4c36904aa..e057d841d023 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/bcm2835-camera.c
@@ -1065,11 +1065,12 @@ static int mmal_setup_components(struct bm2835_mmal_dev *dev,
 		/* Make a further decision on port based on resolution */
 		if (f->fmt.pix.width <= max_video_width &&
 		    f->fmt.pix.height <= max_video_height)
-			camera_port = port =
+			camera_port =
 			    &dev->component[COMP_CAMERA]->output[CAM_PORT_VIDEO];
 		else
-			camera_port = port =
+			camera_port =
 			    &dev->component[COMP_CAMERA]->output[CAM_PORT_CAPTURE];
+		port = camera_port;
 		break;
 	case COMP_IMAGE_ENCODE:
 		encode_component = dev->component[COMP_IMAGE_ENCODE];
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1365_a3385db5a4318bc65789367585d4988fcc2b5033.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1365_a3385db5a4318bc65789367585d4988fcc2b5033.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
